calendars

```diff
@@ -0,0 +1,173 @@
+----- Calendrical issues -----
```

Comment:
This file contains notes on calendrical system adoption from various countries.  The "Eternal Calendar" in Russia is interesting, as it is challenged via an email message.  The issues seems to be left unresolved.

Framed as a more detailed extension of theory.html